tests/conftest.py

```diff
@@ -39,6 +40,11 @@ async def client_test():
 
         # cleanup the test db collections
         await CoreMetadataDOC.find(with_children=True).delete()
+        user = await User.find_one(
+            User.access_token == TEST_ACCESS_TOKEN, fetch_links=True
+        )
+        user.submissions.clear()
+        await user.save(link_rule=WriteRules.WRITE)
```

Comment:
Are the submission documents deleted as well or just the references from the user to the submissions deleted?

Reply:
Since this is a db cleanup function, could you update it to clean up the dangling submissions?  Not a big deal, because it is just for testing.